code/modules/mob/living/carbon/human/human_mob.dm

```diff
@@ -252,19 +252,19 @@
 			if(bomb_armor)
 				brute_loss = 30 * (2 - round(bomb_armor * 0.01, 0.05))
 				burn_loss = brute_loss				//Damage gets reduced from 120 to up to 60 combined brute+burn
-			if(check_ear_prot() < HEARING_PROTECTION_TOTAL)
-				Deaf(2 MINUTES)
+			if(ears && check_ear_prot() < HEARING_PROTECTION_TOTAL)
 				ears.receive_damage(30)
+			Deaf(2 MINUTES)
```

Comment:
You probably want these under the hearing protection blocks, indented another level.

Reply:
As explained on discord, that'd make slimepeople and others without ears immune to being deaf, making this a tweak